Sdk.zig

```diff
@@ -18,11 +18,9 @@ fn sdkRoot() *const [sdkRootIntern().len]u8 {
 
 // linux-x86_64
 pub fn toolchainHostTag() []const u8 {
-    comptime {
-        const os = builtin.os.tag;
-        const arch = builtin.cpu.arch;
-        return @tagName(os) ++ "-" ++ @tagName(arch);
-    }
+    const os = builtin.os.tag;
+    const arch = builtin.cpu.arch;
+    return (comptime if (std.mem.eql(u8, @tagName(os), "macos")) "darwin" else @tagName(os)) ++ "-" ++ @tagName(arch);
```

Comment:
android uses "darwin" tag instead of "macos" as zig does.

Reply:
I suspect you're the first mac use to try this out! Thank you :)